source/_partials/navbar.html

```diff
@@ -127,16 +127,13 @@
     window.addEventListener("load", setActivetrail);
   </script>
 {% endif %}
-{% if page.edtpages%}
+{% if page.edt%}
```

Comment:
The conditional is looking for special_value keys, not custom content types.

The guide's front matter declares `edt: true` so that's what we want here, the special_value key.

https://sculpin.io/documentation/layouts/